### ISSUE TITLE
Support for cinnamon's cinnamenu applet

### ISSUE THREAD

**Issue by fredcw** [EliverLara/Sweet#149]
The class `.menu-category-button-hover{}` in cinnamon.css is used by [cinnamenu](https://cinnamon-spices.linuxmint.com/applets/view/322) applet but at the moment it's bright red.

```
.menu-category-button-hover {
    background-color: red;
    border-radius: 2px; }
```
![Screenshot from 2021-04-24 00-42-05](https://user-images.githubusercontent.com/58893963/115939590-7a5e9280-a496-11eb-89d8-b24719897cbe.png)

Nice theme btw.

**Comment by EliverLara** [comment]
Fixed with 3257037

**Comment by fredcw** [comment]
Sweet.
